app/models/policy.rb

```diff
@@ -7,6 +7,15 @@ class Policy < ApplicationRecord
   validates :name, :version, :account_id, :schema, presence: true
   validate :belongs_to_a_tenant
   validate :validate_schema_specification
+  serialize :schema, ActiveRecord::Coders::JSON
+
+  # Overriding attribute but that is OK
+  def schema=(value)
+    json = value.is_a?(String) ? ActiveRecord::Coders::JSON.load(value.strip) : value
```

Comment:
I think that what I said in https://github.com/3scale/porta/pull/676#discussion_r262194687 is valid for this version too. The equivalent now would be:
```suggestion
    json = ActiveRecord::Coders::JSON.load(value.to_s.strip)
```

Reply:
`{foo: :bar}.to_s` == ""{:foo=>:bar}""`

Reply:
Ok, I get you know 👍